src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    enum Colors { WHITE, YELLOW, ORANGE, RED, MAGENTA, GREEN, BLUE, PINK, BLACK }
```

Comment:
let's create an enum as a separate class

Reply:
in new file?